core/cli.py

```diff
@@ -56,49 +56,13 @@ def seed_test():
     def drop():
         """CLI command to drop all data from the db.
 
-        NOTE: This is not compatible with SQLite.
-
         Example usage:
             flask drop
         """
-        if "sqlite" in current_app.config["SQLALCHEMY_DATABASE_URI"]:
-            print("Drop Failed: Not compatible with SQLite.")
-            return
 
         with current_app.app_context():
             db.session.commit()
             db.drop_all()
             db.create_all()
 
         print("Database data dropped.")
-
-    @app.cli.command("erd")
```

Comment:
Are we happy to remove this? Do we need a todo to restore?

Reply:
happy to remove. You need lots of 3rd party non-python libraries for this to work and it's just not easily maintainable, and currently not possible on our locked-down machines anyway. Now we have PSQL, we can get an ERD directly from PGadmin